Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2822,16 +2822,16 @@ - (NSUUID *)_enableTextIndicatorStylingAfterElementWithID:(NSString *)elementID
     if (!uuid)
         return nil;
 
-#if ENABLE(UNIFIED_TEXT_REPLACEMENT)
+#if ENABLE(WRITING_TOOLS)
     _page->enableTextIndicatorStyleAfterElementWithID(elementID, *uuid);
 
 #if PLATFORM(IOS_FAMILY)
     [_contentView addTextIndicatorStyleForID:nsUUID.get() withStyleType:WKTextIndicatorStyleTypeInitial];
-#elif PLATFORM(MAC) && ENABLE(UNIFIED_TEXT_REPLACEMENT_UI)
```

Comment:
Are you just dropping UNIFIED_TEXT_REPLACEMENT_UI?

Reply:
I think it's okay because it's under `ENABLE(WRITING_TOOLS)`?